recipes/alex/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "alex" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ghc
+    - zlib
```

Comment:
remove zlib and gmp as the host section is enough.
ghc and cobal can be moved to host, hopefully